packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.container.js

```diff
@@ -141,8 +141,12 @@ export class MyAccountOverlayContainer extends PureComponent {
 
         if (isPasswordForgotSend !== currentIsPasswordForgotSend) {
             stateToBeUpdated.isPasswordForgotSend = isPasswordForgotSend;
-            // eslint-disable-next-line max-len
-            showNotification('success', __('If there is an account associated with the provided address you will receive an email with a link to reset your password.'));
+
+            if (!isOverlayVisible) {
+                showNotification('success',
```

Comment:
if you decide to split function call into multiple lines please use pattern:
```js
functionName(
    parameterOne,
    parameterTwo,
    // Comment
    parameterThree
);

// OR

// Comment
functionName(parameterOne, parameterTwo, parameterThree);
````

Reply:
Done